### PR TITLE
PP-12092: Updates recurring payments details

### PR DESCRIPTION
**Pull request by NathanD-GDS**
### Context
Since the initial release of the recurring payments documentation, there’s been a few additions that aren’t captured on the '[Take recurring payments](https://docs.payments.service.gov.uk/recurring_payments)' page or in our API reference.

### Changes proposed in this pull request

This PR:

- adds the cancelled_date response attribute to the API reference pages:
  - Search for agreements
  - Get information about an agreement
- explains that agreements can be cancelled through the admin tool
- explains that services could be charged failed fees if they do not ask for updated card detrails from users
- clarifies the submitted status (ND: do we want to advise how long a payment should be in submitted?)
- mentions that users get an email when they make their first payment (ND: is this their initial set up payment or their first recurring payment?)

### CODE REVIEW

File: source/recurring_payments/index.html.md.erb [hunk 1]
```diff
@@ -182,7 +182,9 @@ Read more about creating payments in [our guidance for taking payments](/making_
 
 ### Update a user’s recurring payment card details
 
-You can update the card used to take recurring payments. This overrides the existing card details. You should do this if a user’s existing card is going to expire soon, or if a user tells you they want to pay with a different card.
+You can update the card used to take recurring payments. This overrides the existing card details. You should do this if a user’s existing card is going to expire soon, or if a user tells you they want to pay with a different card.
```
Comment (charleemg) [inline source/recurring_payments/index.html.md.erb:185]: I don't know if you can reword this a little to make it clear who is doing the using? Even though it's pretty obvious anyway... but maybe something simple like "You can update the card a user will use to take recurring payments". That sounds a bit clunky, though.

File: source/recurring_payments/index.html.md.erb [hunk 2]
```diff
@@ -182,7 +182,9 @@ Read more about creating payments in [our guidance for taking payments](/making_
 
 ### Update a user’s recurring payment card details
 
-You can update the card used to take recurring payments. This overrides the existing card details. You should do this if a user’s existing card is going to expire soon, or if a user tells you they want to pay with a different card.
+You can update the card used to take recurring payments. This overrides the existing card details. You should do this if a user’s existing card is going to expire soon, or if a user tells you they want to pay with a different card. 
+
+You could be charged failed fees if you try to take payments from an expired card.
```
Comment (charleemg) [inline source/recurring_payments/index.html.md.erb:187]: Is "failed fees" a common phrase? Is it "failed payment fees" or something else more descriptive? You'd know probably know way better than me, but just flagging I had to think twice.

**Review by charleemg — APPROVED** [review]

File: source/recurring_payments/index.html.md.erb (hunk 1 above)
Comment (NathanD-GDS) [reply source/recurring_payments/index.html.md.erb:185]: @charleemg Good point! How about:

> You can update the card that GOV.UK Pay uses to take recurring payments from users.

File: source/recurring_payments/index.html.md.erb (hunk 2 above)
Comment (NathanD-GDS) [reply source/recurring_payments/index.html.md.erb:187]: Agreed - will push a fix now.

File: source/recurring_payments/index.html.md.erb (hunk 1 above)
Comment (charleemg) [reply source/recurring_payments/index.html.md.erb]: Perfect! Love it

**Pull request merged**